README.md

```diff
@@ -46,6 +52,52 @@ Then install the dependencies with
 pip install -r requirements.txt
 ```
 
+### Option 2: Use Pipenv to Manage Your Python Virtual Environment
+
+This workflow is better for those looking to contribute to EDGAR.
+
+<details>
+  <summary>Click to expand</summary>
+
+[Pipenv](https://pipenv.pypa.io/en/latest/) is a Python virtualenv management tool.
+It automatically creates and manages a virtualenv for the project, handles adding and removing packages,
+and generates deterministic builds via `Pipfile.lock`.
+
+The recommended Python version for this project is Python 3.12, which you can download from https://www.python.org/downloads/
+
+You can install and activate your Pipenv managed virtual environment by running the following:
+
+```shell
+pip3.12 install pipenv  # Installs Pipenv
+pipenv install  # Installs all dependencies
```

Comment:
Should this be `pipenv install --dev`? As otherwise black is not installed.

Reply:
Good eye. Yes, you are correct. Addressed here: 93bc4b1